universal-login-react/src/ui/UFlow/Devices/ConnectedDevicesItem.tsx

```diff
@@ -31,6 +32,24 @@ export const ConnectedDevicesItem = ({devicesAmount, deviceInfo, publicKey, depl
     }
   };
 
+  const isCurrentDevice = () => utils.computeAddress(deployedWallet.privateKey) === publicKey;
```

Comment:
Extract `utils.computeAddress(deployedWallet.privateKey)` to be a prop of `DeployedWallet`

Reply:
👍